modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -1,20 +1,19 @@
 = Manage Disk Space
-:description: Redpanda provides several ways to manage disk space to ensure the production stability of the cluster.
+:description: Redpanda provides several ways to manage disk space to ensure the stability of a cluster.
 :page-aliases: deployment:well-known-io.adoc, \
 deployment:performance-storage-tuning.adoc, \
 reference:well-known-io.adoc, \
 deployment:disk-utilization.adoc
 
-Redpanda provides several ways to manage disk space to ensure the production stability of the cluster. You can:
+It is important to manage the local disk space in Redpanda to ensure the stability of a cluster. If a node's disk space reaches a critically low level then Redpanda will block clients from writing new data, and furthermore, the Redpanda process will terminate if a node runs out of disk space altogether. This will obviously have a significant impact on performance as client connections and topic data concentrates on fewer nodes.
 
-* Configure message retention and a log cleanup policy.
-* Configure storage threshold properties, and set up alerts to notify you when free disk space reaches a certain threshold.
-* Use xref:./continuous-data-balancing.adoc[Continuous Data Balancing] to automatically distribute partitions across nodes for more balanced disk usage.
-* Create a ballast file to act as a buffer against an out-of-disk outage.
+Redpanda provides several ways to manage disk space, with varying degrees of flexibility and control over which data is removed from local disk and when.
 
-If free disk space reaches a critically low level, Redpanda blocks clients from producing.
-
-When a Redpanda node runs out of disk space, it terminates. This causes leader reassignment and then replication and rebalancing, impacting performance. If more nodes in the cluster fill up and terminate, traffic concentrates in fewer nodes, also impacting performance.
+* Starting with space management, which when used in conjunction with xref:../tiered-storage.adoc[Tiered Storage], treats the local disks as a cache.
+* For more granularity, you can configure topic-level retention policies to manage log cleanup based on partition size or age.
+* Configurable storage thresholds allow you to set up alerts to notify you when disk space is running out.
+* xref:./continuous-data-balancing.adoc[Continuous Data Balancing] can be used to ensure well balanced disk usage across the cluster.
+* A ballast file can be created to allow fast recovery from a full disk.
 
 == Configure message retention
```

Comment:
@jrkinley: there's an existing typo in the following paragraph. Would you please change "...while Redpanda Cloud topics retains 6 hours of data" to "...while Redpanda Cloud topics retain 6 hours of data."

Reply:
And I'm sorry, but also in this first paragraph, would you please change "Proper configuration of message retention properties for your use case will help prevent this from happening." to "Proper configuration of message retention properties for your use case can prevent this from happening."